src/pages/questionPage.js

```diff
@@ -2,32 +2,48 @@
 import {
   NEXT_QUESTION_BUTTON_ID,
   USER_INTERFACE_ID,
+  SKIP_THE_QUESTION,
+  ANSWERS_LIST_ID,
+  NEXT_QUESTION_BUTTON_ID
 } from '../constants.js';
 import {createQuestionElement} from '../views/questionView.js';
-import {createAnswerElements} from '../views/answerView.js'
+import {ShowRightAnswer, createAnswerElements} from '../views/answerView.js'
 import { quizData } from '../data.js';
 
 export const initQuestionPage = () => {
   
-  const currentQuestion = quizData.questions[quizData.currentQuestionIndex];
-  const userInterface = document.getElementById(USER_INTERFACE_ID);
+  const currentQuestion = quizData.questions[quizData.currentQuestionIndex]
+  const userInterface = document.getElementById(USER_INTERFACE_ID)
   userInterface.innerHTML = '';
-   
-  const questionElement = createQuestionElement(currentQuestion.text);
-  userInterface.appendChild(questionElement);  
+  
+  const questionElement = createQuestionElement(currentQuestion.text)
+  userInterface.appendChild(questionElement); 
+  
+  const answersListElement = document.getElementById(ANSWERS_LIST_ID);
 
-  const answerElement = createAnswerElements();
+  const unableTheButton = document.getElementById(NEXT_QUESTION_BUTTON_ID)
```

Comment:
You should use SKIP_THE_QUESTION here, not NEXT_QUESTION_BUTTON_ID

Reply:
This one here to unable the next question button not to skip the question that's why I used NEXT_QUESTION_BUTTON_ID

Reply:
because I give attribute disable for the next question button.

Reply:
Oh yes, I see - in that case the variable should be called `nextQuestionButton`.